kci_test

```diff
@@ -284,8 +284,22 @@ class cmd_submit(Command):
             try:
                 job_id = lab_api.submit(path)
                 print("{} {}".format(job_id, path))
+            except requests.exceptions.HTTPError as e:
+                print(f'HTTPError: {e}')
```

Comment:
Do we still need this with the extra prints we're now adding below?

Reply:
Yes, in some cases it might be non-json reply gateway error (then will go over Reply body), might be truncated/broken json, and non-HTTPError (tcp connection failure, DNS failure). Better to have max verbose errors (considering risks, for example to not leak secrets).
But what i forgot - to set res/return code, added now.